### ISSUE TITLE
An ADA issue: the keyboard focus is not visible for the home page's carousel banner

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#23]
[upwork.com/ab/f/contracts/20874278](https://www.upwork.com/ab/f/contracts/20874278)

![01](https://user-images.githubusercontent.com/83204/47472734-90d9f380-d831-11e8-9332-438a84375de3.png)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.1.4](https://github.com/kitchencollection-com/core/releases/tag/0.1.4).
